### PR TITLE
fix: slack > auto translate event

### PR DESCRIPTION
**Pull request by huglx**
- Fixed the bug where users don't receive messages during auto translation.
- Added tests for Slack integration with auto translation.
- Moved `MachineTranslationTest` to a common package for use in Slack integration tests.

### CODE REVIEW

**Comment by huglx** [comment]
@JanCizmar,I had to disable email verification in the e2e login tests because it sometimes breaks the test.

File: e2e/cypress/e2e/security/login.cy.ts [hunk 1]
```diff
@@ -55,6 +56,7 @@ context('Login', () => {
   });
 
   it('login with github', () => {
+    disableEmailVerification();
```
Comment (JanCizmar) [inline e2e/cypress/e2e/security/login.cy.ts:59]: This looks kinda unrelated to the issue.

**Review by JanCizmar — APPROVED** [review]
